### ISSUE TITLE
Spectrum Device causes script error in ACE arsenal

### ISSUE THREAD

**Issue by Schwaggot** [acemod/ACE3#7257]
**Mods (complete and add to the following information):**
- **Arma 3:** `1.96.146070` [29.10.19 update]
- **CBA:** `3.12.2`
- **ACE3:** `3.12.6`

**Description:**
Script error when selecting Spectrum Device as handgun in ACE arsenal after today's update.

**Steps to reproduce:**
- Go to ACE arsenal
- equip Spectrum Device as handgun

**Where did the issue occur?**
- ACE Arsenal

**Log Files:**
```
20:28:33 Error in expression <{
((_compatibleMagazines select _index) select _subIndex) pushBackUnique (config>
20:28:33   Error position: <select _subIndex) pushBackUnique (config>
20:28:33   Error Zero divisor
20:28:33 File z\ace\addons\arsenal\functions\fnc_fillRightPanel.sqf..., line 83
```

**Comment by jonpas** [comment]
Already fixed in #7256.

**Comment by jonpas** [comment]
Can keep open until merge. :)
